CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+# Version 0.0.0 (YYYY-MM-DD) - In Progress
+
+### Added
+### Changed
```

Comment:
**suggestion** would recommend changing this line:
https://github.com/Labelbox/labelbox-python/blob/3b375d49b5e9ade23151e717876a4e4f3f312881/labelbox/client.py#L614

to include queue_mode in the example:
>>> project = client.create_project(name="<project_name>", description="<project_description>", queue_mode=QueueMode.Batch)